strawberry-daiquiri.json

```
@@ -0,0 +1,56 @@
+{
+    "name": "Strawberry Daiquiri",
+    "description": "A frosty blender full of strawberry-citrus slush. You will enjoy this with or without the rum!",
+    "github": "Kash2000",
+    "ingredients": [
+        {
+            "quantity": "6",
+            "measure": "cups",
+            "ingredient": "ice"
+        },
+        {
+            "quantity": "1/2",
+            "measure": "cup",
+            "ingredient": "lemon juice"
+        },
+        {
+            "quantity": "1/2",
+            "measure": "cup",
+            "ingredient": "white sugar"
+        },
+        {
+            "quantity": "3/4",
+            "measure": "cup",
+            "ingredient": "rum"
+        },
+        {
+            "quantity": "4",
+            "measure": "ounces",
+            "ingredient": "frozen strawberries"
+        },
+        {
+            "quantity": "1/4",
+            "measure": "cup",
+            "ingredient": "lemon-lime flavoured carbonated beverage"
+        },
+        {
+            "quantity": "1/8",
+            "measure": "cup",
+            "ingredient": "lime juice"
+        },
```

```suggestion
        }
```

It's failing when it moves into recipes because it was invalid json due to this comma. It was passing before because the project doesn't test/scan the files at the project root. If you remove this comma, you should be able to move it back to `src/recipes/` again.